src/Microsoft.DotNet.Wpf/src/System.Xaml/System/Xaml/Schema/TypeBits.cs

```diff
@@ -41,14 +41,6 @@ internal enum BoolMemberBits
         Directive    = Default,
         AllValid     = 0xFFFF << 16
     }
-
-    // Use this instead of a Nullable<bool> when a single-word read is needed for thread safety
```

Comment:
What is the impact on thread safety of deleting this?

Reply:
/cc @mikedn

Reply:
It seems like the [most recent comment](https://github.com/dotnet/wpf/issues/218#issuecomment-452104173) implies this is a breaking change... am I reading that right? An explicit "this change is OK" or "this change is not OK" would be very helpful.

Reply:
Well, I'm not familiar with this code so I can only assume that the comment is not out of date or something and that it is indeed supposed to be thread safe. In that case, yes, this change is not safe to make.

Reply:
Thank you for the additional clarification.

Reply:
@mikedn yeah thanks for the clarification. I agree, this does seem to be a breaking change.

Reply:
Closing :)